README.md

```diff
@@ -94,3 +148,37 @@ jobs:
       # Use the standard extra args and ignore spread tests
       trivy-extra-args: '--skip-dirs "tests/spread/**"'
 ```
+
+## Python test runner
+
+The Python test runner workflow uses GitHub workflows and `uv` to run Python tests in
+several forms. It:
+
+#. Runs unit tests across multiple platforms and Python versions.
+#. Runs unit tests on Ubuntu with the oldest supported python version and uv resolution
+set to `lowest`.
+#. Integration tests across multiple platforms and Python versions.
+#. Uploads coverage for unit tests as artefacts.
+#. Runs TICS analysis on unit tests
+
+In order to do so, it expects the following `make` targets:
+
+- `setup-tests`: Configures the system, installing any other necessary tools.
+- `coverage`: Runs unit tests with test coverage
```

Comment:
this target sounds weird in comparison to the others

Reply:
Now's our chance to change it. `test-coverage`?

Reply:
I was going to propose that, but there was a proposal to discuss what coverage means by the team too. So maybe we wait until that discussion tomorrow

Reply:
Based on team discussion I've:

1. Renamed to `test-coverage`
2. Distinguished instead based on fast and slow tests (and updated the starbase PR with a sample)
3. Updated the README appropriately.